minecraft/listener.go

```diff
@@ -172,6 +178,24 @@ func (listener *Listener) Disconnect(conn *Conn, message string) error {
 	return conn.Close()
 }
 
+// AddResourcePack adds a new resource pack to the listener's resource packs.
+// Note: This methods will not update resource packs for active connections.
```

Comment:
I'd just make this say "This method" rather than "This methods", same for RemoveResourcePack